config/master.d/master.conf

```diff
@@ -4,6 +4,7 @@ interface: 0.0.0.0
 event_return: mysql
 presence_events: True
 worker_threads: 10
+timeout: 20
```

Comment:
https://docs.saltstack.com/en/latest/topics/troubleshooting/master.html#commands-time-out-or-do-not-return-output suggests and https://docs.saltstack.com/en/latest/topics/troubleshooting/master.html#salt-master-doesn-t-return-anything-while-running-jobs suggest this is exactly the timeout parameter I'm looking for?